### DISCUSSION TITLE
List of colors

### DISCUSSION THREAD

**Discussion by EvokerKing** [Textualize/rich#441] [Q&A] [answered]
## I am working on a project but need a list of all the colors I can use.

I am making a project where it stores and reads `.json` files in Python with different styling. I want to have styles for each color, but need a list of colors. Can you please link me to one or create one in your answer below?

**Comment by willmcgugan** [comment, marked as answer]
Like this?
 https://rich.readthedocs.io/en/latest/appendix/colors.html
